samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java

```diff
@@ -35,7 +35,7 @@
 @InterfaceStability.Unstable
 public interface ApplicationRunner {
 
-  String RUNNER_CONFIG = "job.runner.class";
+  String RUNNER_CONFIG = "app.runner.class";
```

Comment:
@prateekm that was my suggestion from another PR. Currently, w/ the introduction of the concept of application, we have three different scopes now: application -> job -> task. I chat w/ @navina and @jmakes already. I will let them to confirm.